### DISCUSSION TITLE
Gopro 12 timelapse and daily upload

### DISCUSSION THREAD

**Discussion by Prontoolsen** [gopro/labs#646] [Q&A]
Hi, 

Can some help me with the correct settings?

I am trying to do a timelapse photo, linear, 30min interval, running for 9 hours daily starting at 23.00 every night for two weeks. I also would have love auto upload to the cloud after every recording. I have power available at site.

I tried using this https://gopro.github.io/labs/control/dailytl/

But it only starts recording a movie, not timelapse foto.

The session is planned to start in 10 hours..Help!

/J

**Comment by dnewman-gpsw** [comment]
Movie is better for uploads, that is why is it the default, as it is way more efficient.
 
The default command: **!22:30NmNLp.1800!S!32460E!U!1R**   (mNL <- mode night-lapse video)

for NLP you need both mNP and the present to exist on camera (it doesn't by default.)

So the command would be: **!22:30NmNPp.1800!S!32460E!U!1R**
